bgpd/bgp_snmp_bgp4v2.c

```diff
@@ -806,7 +806,7 @@ static struct variable bgpv2_variables[] = {
 	 RONLY,
 	 bgpv2PeerTable,
 	 6,
-	 {1, 2, 1, BGP4V2_PEER_INSTANCE, 2, 2}},
+	 {1, 2, 1, BGP4V2_PEER_INSTANCE, 1, 2}},
```

Comment:
Probably that's fine, I just remember that I sticked to what Cisco/Arista do without paying too much checking on how they are translated.

Reply:
from https://datatracker.ietf.org/doc/html/draft-ietf-idr-bgp4-mibv2-11

bgp4V2PeerEntry OBJECT-TYPE
        SYNTAX     Bgp4V2PeerEntry
        MAX-ACCESS not-accessible
        STATUS     current
        DESCRIPTION
            "Entry containing information about the connection with
             a remote BGP peer."
        INDEX {
            bgp4V2PeerInstance,
            bgp4V2PeerRemoteAddrType,
            bgp4V2PeerRemoteAddr
        }


 bgp4V2PeerInstance OBJECT-TYPE
        SYNTAX     Unsigned32 (1..4294967295)
        MAX-ACCESS not-accessible
        STATUS     current
        DESCRIPTION
            "The routing instance index.

             Some BGP implementations permit the creation of
             multiple instances of a BGP routing process. An
             example includes routers running BGP/MPLS IP Virtual
             Private Networks.

             **Implementations that do not support multiple
             routing instances should return 1 for this object."**
        ::= { bgp4V2PeerEntry 1 }


in my opinion, the MIB description is not human readable